addons/purchase/report/purchase_report.py

```diff
@@ -147,3 +155,46 @@ def _group_by(self):
                 po.id
         """
         return group_by_str
+
+    @api.model
+    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
```

Comment:
seems complex to get the average days. Why not implementing it the same way as Sales ? You can check https://github.com/odoo/enterprise/blob/master/sale_enterprise/report/sale_report.py and the corresponding view

Reply:
Sales was calculating the wrong aggregated value (I was originally calculating it the same way, but the error became obvious with the new custom purchase dashboard that has the same value correctly calculated). I propose a similar hack for the sales report: https://github.com/odoo/enterprise/pull/8945/commits/b8762d99566abc2dc021dbc0b41135bc3e37b407

Reply:
Ok thanks for the explanation